roles/dhcp-server/templates/boxes.conf.j2

```diff
@@ -0,0 +1,20 @@
+# /etc/dnsmasq.d/boxes.conf
+#
+
+# defaults:
+# drive
+# tasks
+
+dhcp-option-force=209,"partman-auto/disk=/dev/{{default_install_dev}} grub-installer/bootdev=/dev/{{default_install_dev}} tasks={{default_install_tasks}}"
```

Comment:
Why force?

Reply:
I can't say for certain that it is needed, but given the example I'd just go with it.

"This works in exactly the same way as --dhcp-option except that the option will always be sent, even if the client does not ask for it in the parameter request list. This is sometimes needed, for example when sending options to PXELinux. "  
http://www.thekelleys.org.uk/dnsmasq/docs/dnsmasq-man.html